about.html

```diff
@@ -0,0 +1,222 @@
+                <h2>Hello, welcome to</h2>
+            </div>
+            <div class="headline-title">
+                <h1>AFRICAN MOVIES<br />FESTIVAL 2023</h1>
+            </div>
+            <div class="headline-paragraph">
+                <p>
+
+                    Africa movies festival is an African forum. Our mission is to unify the global African film making
+                    and build a future of movie industry that embodies our culture, connection, and community. We are
+                    creating a world where the African film making is leading film making industry in their communities.
+                    We welcome you to our networking dinner in Rwanda
+
+                </p>
+            </div>
+            <div class="headline-time">
+                <p>Please contact us for further questions about Africam Movies Festival</p>
+            </div>
+            <div class="headline-venue">
+
+                <h5>afrimovies@africa.com</h5>
+            </div>
+        </div>
+    </section>
+    <section class="about-logo">
+        <div class="about-logo-container">
+            <div class="about-logo-title">
+                <h2>African Movies Festival 2023 Logo</h2>
+                <hr class="line">
+            </div>
+            <div class="desk-about-logo-title">
+                <h2>African Movies Festival 2023 Logo</h2>
+                <hr class="line">
+            </div>
+            <div class="about-logo-paragraph">
+                <p>
+                    African Movies Festival introduced a new logo -: an expression of the diversity in our people and
+                    film makers. We believe that Africa is one, and the only language that brings film makers together
+                    is "action." Our mission is to unify African movie makers and build a future of movie industry that
+                    embodies our culture, connection, and community.
+                </p>
+            </div>
+            <div class="about-logo-image">
+                <img src="./images/download.jpg" alt="">
+            </div>
+        </div>
+
+    </section>
+
+    <section class="stories">
+        <div class="stories-container">
+            <div class="story-title">
+                <h2>See our previous events and projects</h2>
+                <hr class="line">
+            </div>
+            <div class="story-paragraph">
+                <p>African Movies endeavors to make Africa the Film making center </p>
+            </div>
+
+            <!-- <div class="story-images-container">
+                <div class="story-images">
+                    <h4 class="year">2023</h4>
+                    <p class="para">Introduce film making in campus</p>
+                </div>
+                <div class="story-images2">
+                    <h4 class="year">2023</h4>
+                    <p class="para">Introduce film making in campus</p>
+                </div>
+            </div> -->
```

Comment:
- [ ] Please remove all commented code from your codebase because it's a bad practice to commit commented-out code. That's what version control is for. If you need that code back, just fetch it from the repo :smile: